src/frontend/index.ts

```diff
@@ -272,6 +276,9 @@ export async function createFlowrWindow(flowrStore: Store<IFlowrStore>): Promise
       app.relaunch()
       app.quit()
     },
+    setFlowrMonitoringTimer:(evt: IpcMainEvent, monitorTimer: number) => {
+      flowrStore.set('flowrMonitoringTime',monitorTimer)
```

Comment:
Just so you know: most values contained in both flowr and player stores should be configurable from flowr-admin (Settings -> Desktop Settings).
Could you please create a ticket so we add the missing values ?